server/src/main/scala/actors/ArrivalsActor.scala

```diff
@@ -152,11 +150,11 @@ abstract class ArrivalsActor(now: () => SDateLike,
   def consumeDiffsMessage(message: FlightsDiffMessage): Unit
 
   def consumeRemovals(diffsMessage: FlightsDiffMessage): Unit = {
-    logRecoveryMessage(s"Consuming ${diffsMessage.removals.length} removals")
-    restorer.removeLegacies(diffsMessage.removalsOLD)
-    restorer.remove(diffsMessage.removals.map(uam =>
-      UniqueArrival(uam.getNumber, uam.getTerminalName, uam.getScheduled)
-    ))
+    if (diffsMessage.removalsOLD.nonEmpty)
```

Comment:
does this empty check achieve anything, seeing as an empty list would just be a no-op?

Reply:
:+1: